ccloader/js/ccloader.js

```diff
@@ -255,6 +257,12 @@ export class ModLoader {
 			case 'crosscode':
 				depVersion = this.ccVersion;
 				break;
+			case 'post-game':
```

Comment:
Please don't hardcode a list of extensions here. The `default` branch should check if `depName` is in the extensions array, assign `depVersion` to the game version if that's the case, otherwise check if `depName` refers to a mod.

Reply:
I agree, if you want a list of known (missing) extensions you should use add array of names either above the class or in the constuctor.